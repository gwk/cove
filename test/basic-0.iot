{
  'cmd': '$PROJ/cove.py test/basic.py 0',
  'links': 'test',
}
