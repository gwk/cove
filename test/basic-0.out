
cove report:

__main__: test/basic.py:
[30m   5 [0m  [0m
[30m   6 [0m  def main():[0m
[30m   7 [0m    arg = int(argv[1])[0m
[30m   8 [0m    if arg:[0m
[30m   9 [31m!     called_one()[0m
[30m  10 [31m!     called_multi()[0m
[30m  11 [31m!     ignored_but_called_one()[0m
[30m  12 [31m!     ignored_but_called_multi()[0m
[30m  13 [0m  [0m
[30m  14 [0m  # Padding lines to force hunk split.[0m
[30m  16 [0m  # Padding lines to force hunk split.[0m
[30m  17 [0m  # ^[0m
[30m  18 [0m  # ^[0m
[30m  19 [0m  [0m
[30m  20 [31m% def called_one(): return None[0m
[30m  21 [0m  [0m
[30m  22 [0m  def called_multi():[0m
[30m  23 [31m!   return None[0m
[30m  24 [0m  [0m
[30m  25 [0m  [0m
__main__: test/basic.py: 38 lines; 18 traceable; 4 ignored; 0 IGNORED but covered; 6 NOT COVERED.
